manifests/deployment.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kepler-model-server
+  labels:
+    app: sustainable-computing.io
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: sustainable-computing.io
+  template:
+    metadata:
+      labels:
+        app: sustainable-computing.io
+    spec:
+      containers:
```

Comment:
also add a persistent storage so that the models are persisted and can be reused.